plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -776,6 +779,13 @@ public boolean configure(final String name, final Map<String, Object> params) th
             _localStoragePath = "/var/lib/libvirt/images/";
         }
 
+        /* Directory to use for Qemu sockets like for the Qemu Guest Agent */
+        _qemuSocketsPath = new File("/var/lib/libvirt/qemu");
+        String _qemuSocketsPathVar = (String)params.get("qemu.sockets.path");
+        if (!StringUtils.isNotBlank(_qemuSocketsPathVar)) {
```

Comment:
Minor not: A logical not perform on `StringUtils.isNotBlank` is a awkward.  Please consider refactoring to `if (StringUtils.isBlank(_qemuSocketsPathVar)`.